message_definitions/v1.0/messages.xml

```diff
@@ -2948,6 +2948,49 @@
     <field name="itow"   type="uint32" unit="ms"/>
   </message>
 
+  <message name="SHAPE" id="38">
+    <description>
+      The SHAPE message used to draw shapes onto the Paparazzi GCS.
+      Field name shape is used to define the type of shape i.e. Circle, Polygon, Line, or Text.
+      This is indexed from 0-3 respectively.
+
+      Each shape type has can be given a unique id number.
+      ie. a circle can be defined with id 1 and so can a polygon.
+
+      linecolor and fillcolor take in a color string ie: "red", "blue" 
+      
+      opacity will change the level of transparency of the fill.
+      0 - Transparent
+      1 - Light Fill
+      2 - Medium Fill
+      3 - Opaque
+
+      passing a status of 0 will create or update a shape specified by id and type.
+      passing a status of 1 will delete the shape specified by the id and the type.
+
+      Latarr and lonarr contain the coordinates for the shapes.
```

Comment:
At first reading, the terms `latarr` and `lonarr` was not obvious. I suggest to tell explicitly they are "array of coordinates", and eventually change the name to `lat` `lon` since the type already tell it should be an array.
Also, you should recall that arrays are coma separated values (without spaces).